javascript/node/selenium-webdriver/test/cookie_test.js

```diff
@@ -212,6 +212,53 @@ suite(function (env) {
         assert.equal(actualCookie.sameSite, 'Lax')
       }
     )
+
+    ignore(env.browsers(Browser.FIREFOX, Browser.IE, Browser.SAFARI)).it(
```

Comment:
Shouldn't these tests work in Firefox?

Reply:
I mirrored the original tests, didn't want to change too much. I'll double-check. Maybe it wasn't an option at the time of the original implementation but it is now.

Reply:
You're right, updated.